### PR TITLE
added some styles to the inputs using a class named "text_input" as t…

### PR DESCRIPTION
**Pull request by Andy-Not**
new styles for inputs let me know how you guys feel about them.

### CODE REVIEW

**Comment by vesutiiji** [comment]
the changes pops too much, make it more subtle so that it blends in with everything else on the form.

File: client/styles/styles.css [hunk 1]
```diff
@@ -4,7 +4,15 @@
...
-
+h1{
+    text-align: center;
+    padding: 10px;
+    margin-bottom: 10px;
+    border-radius: 5px;
+    color: white;
+    background-color: #0086E3;
+    box-shadow: 0px 0px 14px 1px rgba(0,0,0,0.52);
+}
```
Comment (dodavid808) [inline client/styles/styles.css]: not sure why this code is here. this was done on your other branch. dont have to repeat what you did onto this branch. focus on the text_input only.

**Review by dodavid808 — CHANGES_REQUESTED** [review]
@Andy-Not see comment.

File: client/styles/styles.css (hunk 1 above)
Comment (Andy-Not) [reply client/styles/styles.css]: got it

**Review by Andy-Not — COMMENTED** [review]
image

**Review by vesutiiji — APPROVED** [review]
i think it's okay.

**Pull request merged**